### PR TITLE
Push new file manager in the demo.

### PR DESCRIPTION
**Pull request by karandatwani92**
### BEFORE - What was wrong? What was happening before this PR?

image

### AFTER - What is happening after this PR?

image

### How did you achieve that, in technical terms?

```
rm -rf resources/views/vendor/elfinder
php artisan backpack:filemanager:install
```

### CODE REVIEW

**Comment by karandatwani92** [comment]
This PR has v3.0.3 [FileManager](https://github.com/Laravel-Backpack/FileManager)

**Comment by pxpm** [comment]
@karandatwani92 what you think if we add the publish command in the `composer.json` `post-update` / `post-install` scripts ?

So every time we deploy code to demo, we re-publish the elfinder assets with: `php artisan vendor:publish --provider="Backpack\FileManager\FileManagerServiceProvider" --tag=views --force`
?

That way we won't forget to re-publish elfinder assets when we change FileManager🙏

**Comment by karandatwani92** [comment]
> @karandatwani92 what you think if we add the publish command in the `composer.json` `post-update` / `post-install` scripts ?
> 
> So every time we deploy code to demo, we re-publish the elfinder assets with: `php artisan vendor:publish --provider="Backpack\FileManager\FileManagerServiceProvider" --tag=views --force` ?
> 
> That way we won't forget to re-publish elfinder assets when we change FileManager🙏

@pxpm Done! Please check & merge🎉

File: composer.json [hunk 1]
```diff
@@ -70,10 +70,12 @@
...
         "post-install-cmd": [
             "Illuminate\\Foundation\\ComposerScripts::postInstall",
-            "php artisan storage:link"
+            "php artisan storage:link",
+            "php artisan vendor:publish --provider=\"Backpack\\FileManager\\FileManagerServiceProvider\" --tag=views --force"
         ],
         "post-update-cmd": [
-            "Illuminate\\Foundation\\ComposerScripts::postUpdate"
+            "Illuminate\\Foundation\\ComposerScripts::postUpdate",
+            "php artisan vendor:publish --provider=\"Backpack\\FileManager\\FileManagerServiceProvider\" --tag=views --force"
```
Comment (pxpm) [inline composer.json:78]:
```suggestion
```

I think it's enough to have it in the `install` cmd. You shouldn't be running `update` in production environment.

**Comment by tabacitu** [comment]
> So every time we deploy code to demo, we re-publish the elfinder assets with: php artisan vendor:publish --provider="Backpack\FileManager\FileManagerServiceProvider" --tag=views --force

Hmm... why would we re-publish the assets on every install? That's a one-and-done deal for projects, right? Since we don't tell people to add this command to their composer.json I don't think we should add it either. It might trick us into pushing breaking changes into FileManager and thinking they're non-breaking.

**Comment by tabacitu** [comment]
> So every time we deploy code to demo, we re-publish the elfinder assets with: php artisan vendor:publish --provider="Backpack\FileManager\FileManagerServiceProvider" --tag=views --force

Hmm... why would we re-publish the assets on every install? That's a one-and-done deal for projects, right? Since we don't tell people to add this command to their composer.json I don't think we should add it either. It might trick us into pushing breaking changes into FileManager and thinking they're non-breaking.

**Comment by pxpm** [comment]
Hey @karandatwani92 sorry for the trouble here, I've mistakenly induced you in error to create the entry in `composer.json` but @tabacitu don't agree with that change. 

Can you please remove those changes and put this PR like you had in the beginning by publishing the views?

Once again, sorry for more work 🙏

**Comment by karandatwani92** [comment]
Master @pxpm It's OK. UNDO is easy.😜

**Pull request merged**